simulation_parameters/Constants.cs

```diff
@@ -186,6 +186,16 @@ public static class Constants
     /// </summary>
     public const float ORGANELLE_GROW_STORAGE_MUST_HAVE_AT_LEAST = 0.0f;
 
+    /// <summary>
+    ///   Cost of moving the rigidity slider by one step in the microbe editor
+    /// </summary>
+    public const int MEMBRANE_RIGIDITY_COST_PER_STEP = 2;
+
+    /// <summary>
+    ///   Number used to convert between the valeu from the rigidity slider and the actual value
+    /// </summary>
+    public const int MEMBRANE_RIGIDITY_SLIDER_TO_VALUE_RATIO = 10;
```

Comment:
Is this used by converting to float anywhere? If so it would make sense to have it as a float here to avoid a bunch of casts.